doc/source/rllib/rllib-replay-buffers.rst

```diff
@@ -0,0 +1,145 @@
+.. _replay-buffer-api-reference-docs:
```

Comment:
Wait, this is the "reference-docs" and the one above is the "references-api-docs", no?